src/components/Select/Select.js

```diff
@@ -80,10 +77,10 @@ export const Select = ({
     let resolvedValues = ''
     // react-select multi select case
     if (Array.isArray(userSelection)) {
-      resolvedValues = userSelection.map((selection) => selection.value)
+      resolvedValues = userSelection?.map((selection) => selection?.value)
```

Comment:
one last check, do you need to add a `.filter(el => el)` to strip any `undefined` from resolvedValues?

Reply:
Absolutely, [90097bb](https://github.com/getethos/ethos-design-system/pull/437/commits/90097bb5dcaf56e3e7983ec3bdefd8cd4eeb6573)

Reply:
Per slack DMs, [d29d345](https://github.com/getethos/ethos-design-system/pull/437/commits/d29d345adbb44c8e803aad270b867f9dc2094784)